front-end/src/components/Post/GovernanceSideBar/Proposals/ProposalVoteInfo.tsx

```diff
@@ -42,6 +44,7 @@ const ProposalVoteInfo = ({ className, proposalId }:  Props) => {
 					setDeposit(formatBnBalance(proposal.balance, { numberAfterComma: 2, withUnit: true }));
 				}
 			});
+			setIsInfoLoading(false);
```

Comment:
I don't think that's done right here. I guess this should be rather in an async function (?)